Ladybird/Qt/Tab.cpp

```diff
@@ -523,28 +523,20 @@ Tab::Tab(BrowserWindow* window, RefPtr<WebView::WebContentClient> parent_client,
         m_page_context_menu->exec(view().map_point_to_global_position(content_position));
     };
 
-    auto* open_link_action = new QAction("&Open", this);
-    open_link_action->setIcon(load_icon_from_uri("resource://icons/16x16/go-forward.png"sv));
-    QObject::connect(open_link_action, &QAction::triggered, this, [this]() {
-        open_link(m_link_context_menu_url);
-    });
-
-    auto* open_link_in_new_tab_action = new QAction("&Open in New &Tab", this);
+    auto* open_link_in_new_tab_action = new QAction("Open Link in New &Tab", this);
     open_link_in_new_tab_action->setIcon(load_icon_from_uri("resource://icons/16x16/new-tab.png"sv));
     QObject::connect(open_link_in_new_tab_action, &QAction::triggered, this, [this]() {
         open_link_in_new_tab(m_link_context_menu_url);
     });
 
-    m_link_context_menu_copy_url_action = new QAction("Copy &URL", this);
+    m_link_context_menu_copy_url_action = new QAction("Copy &Link", this);
```

Comment:
Optional but I think we could specify this as Copy Link Address instead of Copy Link :~)

Reply:
Done.